lib/customer.rb

```diff
@@ -0,0 +1,77 @@
+require_relative 'order'
+require_relative 'menu'
+require_relative 'twilio'
+
+class Customer
+  attr_accessor :name, :phone_number, :order, :selection, :price
+  def initialize(order = Order.new)
+    @name
+    @phone_number
+    @order = order
+    @selection
+    @quantity
+    @price
+  end
+
+  def place_order
+    get_info
+    @order.select_dishes(@selection, @price, @quantity)
+    get_total
+  end
+
+  def get_info
+    get_name
+  end
+
+  def get_name
+    puts "Good evening, can I get your name please?"
+    @name = gets.chomp
+    get_phone
+  end
+
+  def get_phone
+    puts "and what is your phone number, for delivery verification?"
+    @phone_number = gets.chomp.to_i
+    see_menu
+  end
+
+  def see_menu
+    self.order.menu.display.each { |k,v| puts "#{k}:#{v}" }
```

Comment:
Space missing after comma.<br>Do not write to stdout. Use Rails' logger if you want to log.